### PR TITLE
NPS widget appearance updates

### LINKED ISSUE #1124: Alter the NPS collection widget to drive response rate
We critically need an NPS baseline. This is among our most important organizational objectives. It is worth some compromise in user experience best practices to achieve this. It is even worth artificially deflating that baseline by mildly annoying our users if it allows us to collect it!

Given that our active user numbers are low during Beta phase, acquiring enough data to supply a reliable baseline means maximizing our response rate.

We wish to make the following alterations to the NPS collection widget to achieve this

Currently, we ask a generic question like "how are we doing", with an option to dismiss the widget. If a user opens the widget they see 2 qualitative questions.

The numerical NPS question should be surfaced by default. Further, we should remove the option to close the widget without responding to the numeric question.

If the user answers the numeric question, the widget should expand to present the two qualitative questions. At the same time, the user should gain the ability to dismiss the widget without answering the qualitative questions

### PR DESCRIPTION
**Pull request by katevoss**
Fixes #1124 

For testing this, I can force-display the NPS widget for a user on dev. Let me know if you want me to do this.

### CODE REVIEW

File: src/components/NpsSurvey.js [hunk 1]
```diff
@@ -118,23 +118,24 @@ export const NpsSurvey = Utils.connectAtom(authStore, 'authState')(class NpsSurv
...
-            }, 'Submit')
-          ])
-        ]
+      !expanded ? [
+        div({ style: styles.questionLabel }, 'How likely are you to recommend Terra to others?'),
+        div({ style: { display: 'flex', justifyContent: 'space-around', marginBottom: '0.5rem' } }, scoreRadios)
+      ] : [
+        div({ style: styles.questionLabel }, 'How likely are you to recommend Terra to others?'),
+        div({ style: { display: 'flex', justifyContent: 'space-around', marginBottom: '0.5rem' } }, scoreRadios),
+        span({ style: styles.questionLabel }, 'What was the reason for this score? '), span({ style: { ...styles.questionLabel, color: colors.gray[3] } }, '(Optional)'),
```
Comment (breilly2) [inline src/components/NpsSurvey.js:127]: I usually wrap inline elements (span) in a block level element (div) if the rest of the siblings are block level. I don't think it's critical, though. It all rendered correctly for me so it's probably fine in this case.

**Review by breilly2 — APPROVED** [review]
👍 Nice! I had to try it to prove to myself that it worked. I was expecting to see code to expand it in response to selecting a score, but between the `scoreRadios` and the surrounding `Clickable`, everything works perfectly!

**Comment by elizzhao** [comment]
screen shot 2019-01-30 at 1 41 35 pm

On my computer it looks really squished by the 10. Likely due to the scrollbar or dimension differences, but might be worth adding some padding in case?

**Review by elizzhao — APPROVED** [review]
Aside from the potential padding thing, it looks good and works well!

File: src/components/NpsSurvey.js (hunk 1 above)
Comment (katevoss) [reply src/components/NpsSurvey.js:127]: I tried wrapping the `span`s in a `div` and it messed up the display, so I'm going to leave it as is.

**Pull request merged**